manifests/init.pp

```diff
@@ -38,7 +38,7 @@
   $purge_vhost_dir                                               = undef,
   $purge_vdir                                                    = false,
   $serveradmin                                                   = 'root@localhost',
-  $sendfile                                                      = 'On',
+  Enum['On', 'Off', 'on', 'off'] $sendfile                       = 'On',
```

Comment:
I can do that, if wanted.

However, that would be a separate PR, which also changes all occurrences of it.